concept/answer/Numeric.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2020 Grakn Labs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package grakn.core.concept.answer;
+
+/**
+ * A type of Answer object that contains a Number.
+ */
+public class Numeric implements Answer {
+
+    private final Number number;
+
+    public Numeric(Number number) {
+        this.number = number;
+    }
+
+    public Number number() {
+        return number;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == this) return true;
+        if (obj == null || getClass() != obj.getClass()) return false;
+        Numeric a2 = (Numeric) obj;
+        return this.number.toString().equals(a2.number.toString());
```

Comment:
Are we sure of this, @flyingsilverfin ? Is this what we had in the old code? I'm not quite sure this is right.

Reply:
updated to make sure the long and double values are the same.